components/org.wso2.carbon.identity.conditional.auth.functions.elk/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/elk/internal/ElasticFunctionsServiceComponent.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.conditional.auth.functions.elk.internal;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.wso2.carbon.identity.application.authentication.framework.JsFunctionRegistry;
+import org.wso2.carbon.identity.conditional.auth.functions.elk.CallElasticFunction;
+import org.wso2.carbon.identity.conditional.auth.functions.elk.CallElasticFunctionImpl;
+import org.wso2.carbon.identity.core.util.IdentityCoreInitializedEvent;
+
+/**
+ * OSGi declarative services component which handle cookie related conditional auth functions.
+ */
+
+@Component(
+        name = "identity.conditional.auth.functions.elk.component",
+        immediate = true
+)
+public class ElasticFunctionsServiceComponent {
+
+    private static final Log LOG = LogFactory.getLog(ElasticFunctionsServiceComponent.class);
+
+    public static final String FUNC_CALL_ELASTIC = "callElastic";
+
+    @Activate
+    protected void activate(ComponentContext ctxt) {
+        LOG.error("CALLED ACTIVATE");
```

Comment:
Is this an error log?

Reply:
Mistakenly put. Removed. https://github.com/wso2-extensions/identity-conditional-auth-functions/pull/98/commits/8cb261968f2481e6bf3b6f9ecc81862ce796db67